pb-coroutines-retry/src/main/kotlin/io/provenance/coroutines/retry/store/RetryRecordStore.kt

```diff
@@ -5,6 +5,6 @@ import java.time.OffsetDateTime
 interface RetryRecordStore<T> {
     suspend fun select(attemptRange: IntRange, lastAttempted: OffsetDateTime): List<RetryRecord<T>>
     suspend fun getOne(item: T): RetryRecord<T>?
-    suspend fun putOne(item: T, lastException: Throwable? = null, mutator: (RetryRecord<T>) -> RetryRecord<T> = { it })
+    suspend fun putOne(item: T, lastException: Throwable? = null, mutator: RetryRecord<T>.() -> RetryRecord<T> = { this })
```

Comment:
mutator can be of type `RetryRecord<T>.() -> Unit` since the `var` fields are allowing overwrite now (and the default value `{}` instead of `{ this }`)